### PR TITLE
Add --framework_name option

### PR DESCRIPTION
**Pull request by fpgeek**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -10,6 +10,7 @@ marathon_apt_repo: "deb http://repos.mesosphere.com/{{ansible_distribution|lower
 marathon_yum_package: "marathon-{{ marathon_version }}"
 mesosphere_yum_repo: "http://repos.mesosphere.com/el/{{ os_version_major }}/noarch/RPMS/{{ mesosphere_releases[os_version_major] }}"
 
+marathon_framework_name: marathon
```
Comment (ninthnails) [inline defaults/main.yml]: Could you please use the default value used by Marathon, which is `marathon-VERSION` where version is version of marathon of course.

**Comment by ninthnails** [comment]
Build is failing if you haven't noticed. Once fixed, we can go ahead merging.

File: defaults/main.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 marathon_playbook_version: "0.3.2"
-marathon_version: "0.8.1"
+marathon_version: "0.11.1"
```
Comment (ernestas-poskus) [inline defaults/main.yml]: version change must be in a separate commit

File: defaults/main.yml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 ---
 marathon_playbook_version: "0.3.2"
```
Comment (ernestas-poskus) [inline defaults/main.yml]: bump playbook version

**Comment by ernestas-poskus** [comment]
please add tests to .travis.yml

**Comment by ernestas-poskus** [comment]
ping @fpgeek

**Comment by lhoss** [comment]
@fpgeek  you could also set the framework_name  using the new feature added yesterday by https://github.com/AnsibleShipyard/ansible-marathon/pull/28

**Comment by ernestas-poskus** [comment]
Closing due to lack of activity.
